editor/editor_node.cpp

```diff
@@ -5850,14 +5850,31 @@ EditorNode::EditorNode() {
 	about->get_ok()->set_text(TTR("Thanks!"));
 	about->set_hide_on_ok(true);
 	gui_base->add_child(about);
+	VBoxContainer *vbc = memnew(VBoxContainer);
 	HBoxContainer *hbc = memnew(HBoxContainer);
-	about->add_child(hbc);
+	about->add_child(vbc);
+	vbc->add_child(hbc);
 	Label *about_text = memnew(Label);
-	about_text->set_text(VERSION_FULL_NAME "\n(c) 2008-2017 Juan Linietsky, Ariel Manzur.\n");
+	about_text->set_text(VERSION_FULL_NAME + String::utf8("\n\u00A9 2007-2017 Juan Linietsky, Ariel Manzur.\n\u00A9 2014-2017 Godot Engine contributors\n"));
 	TextureRect *logo = memnew(TextureRect);
 	logo->set_texture(gui_base->get_icon("Logo", "EditorIcons"));
 	hbc->add_child(logo);
 	hbc->add_child(about_text);
+	HBoxContainer *author_hbc = memnew(HBoxContainer);
+	vbc->add_child(author_hbc);
+	Label *author1_text = memnew(Label);
+	Label *author2_text = memnew(Label);
+	Label *author3_text = memnew(Label);
+	author_hbc->add_child(author1_text);
+	author_hbc->add_child(author2_text);
+	author_hbc->add_child(author3_text);
+	author1_text->set_h_size_flags(3);
+	author2_text->set_h_size_flags(3);
+	author3_text->set_h_size_flags(3);
+	author3_text->set_v_size_flags(1);
+	author1_text->set_text(String::utf8("Alexander Holland (AlexHolly)\nAlexey Velikiy (jonyrock)\nAndreas Haas (Hinsbart)\nAnton Yabchinskiy (a12n)\nAren Villanueva (kurikaesu)\nAriel Manzur (punto-)\nBastiaan Olij (BastiaanOlij)\nBojidar Marinov (bojidar-bg)\nBłażej Szczygieł (zaps166)\nCarl Olsson (not-surt)\nDana Olson (adolson)\nDaniel J. Ramirez (djrm)\nEmmanuel Leblond (touilleMan)\nFabio Alessandrelli (Faless)\nFerenc Arn (tagcup)\nFranklin Sobrinho (TheHX)\nGeequlim\nGen (dbsGen)\nGeorge Marques (vnen)\nGuilherme Felipe (guilhermefelipecgs)"));
+	author2_text->set_text(String::utf8("Hein-Pieter van Braam (hpvb)\nHubert Jarosz (Marqin)\nIgnacio Etcheverry (neikeq)\nJ08nY\nJohan Manuel (29jm)\nJoshua Grams (JoshuaGrams)\nJuan Linietsky (reduz)\nJulian Murgia (StraToN)\nKostadin Damyanov (Max-Might)\nL. Krause (eska014)\nMarc Gilleron (Zylann)\nMarcelo Fernandez (marcelofg55)\nMariano Javier Suligoy (MarianoGnu)\nMario Schlack (hurikhan)\nMasoud BH (masoudbh3)\nNathan Warden (NathanWarden)\nOvnuniarchos\nPatrick (firefly2442)\nPaul Batty (Paulb23)\nPawel Kowal (pkowal1982)"));
+	author3_text->set_text(String::utf8("Pedro J. Estébanez (RandomShaper)\nRalf Hölzemer (rollenrolm)\nRamesh Ravone (RameshRavone)\nRay Koopa (RayKoopa)\nRémi Verschelde (akien-mga)\nSaracenOne\nThomas Herzog (karroffel)\nV. Vamsi Krishna (vkbsb)\nVinzenz Feenstra (vinzenz)\nZher Huei Lee (leezh)\nZuBsPaCe\n박한얼 (volzhs)\nest31\nmarynate\nmrezai\nromulox-x\nsanikoyes\nyg2f (SuperUserNameMan)"));
```

Comment:
+1 For letting scons generate this.

Also, maybe it would be better to have the list in a `ScrollContainer` instead? The dialog looks quite full already, and I think we're bound to have even more contributors in the future :)
Furthermore, @reduz is planning to get funding via Patreon and put the sponsors names in this dialog, needing even more space.